DSharpPlus/Entities/Channel/Thread/Forum/DefaultReaction.cs

```diff
@@ -40,5 +40,17 @@ public sealed class DefaultReaction
         /// </summary>
         [JsonProperty("emoji_name")]
         public string EmojiName { get; internal set; }
+        
+        /// <summary>
+        /// Creates a DefaultReaction object from an emoji.
+        /// </summary>
+        /// <param name="emoji">The <see cref="DiscordEmoji"/>.</param>
+        /// <returns>Create <see cref="DefaultReaction"/> object.</returns>
+        public static DefaultReaction FromEmoji(DiscordEmoji emoji)
```

Comment:
shouldn't this ideally just be an actual constructor, instead of a static method

Reply:
Done
(went with a static constructor because i tried to copy style from DiscordEmoji but those constructors are all a bit different i guess)

Reply:
small nitpick, this isn't a static constructor, just a static creation method. a static constructor serves to initialize the statics on a type :)